library/src/test/java/com/chuckerteam/chucker/internal/support/OkioUtilsTest.kt

```diff
@@ -0,0 +1,56 @@
+package com.chuckerteam.chucker.internal.support
+
+import com.google.common.truth.Truth
+import okio.Buffer
+import okio.ByteString.Companion.encodeUtf8
+import org.junit.jupiter.api.Test
+import java.nio.charset.Charset
+
+internal class OkioUtilsTest {
+    @Test
+    fun isPlaintext_withEmptyBuffer_returnsTrue() {
+        val buffer = Buffer()
+
+        Truth.assertThat(buffer.isProbablyPlainText).isTrue()
```

Comment:
Nitpick: why don't we import this function for consistency with other tests?

Reply:
Yup, thanks! Saw it as well and forgot to change it 😅 .